### PR TITLE
update slap

### PR DESCRIPTION
**Pull request by Dekriel**

### CODE REVIEW

**Review by Yat-o — CHANGES_REQUESTED** [review]
session isn’t a attribute of our current bot class. And target isn’t a valid name in the command. Revise and commit again.

File: src/cogs/fun.py [hunk 1]
```diff
@@ -141,18 +141,17 @@ async def kiss(self, ctx: Context, member: Member = None):
...
-        )
-
-        await ctx.send(embed=embed2)
+    async def slap(self, ctx, member: Member = None):
+        """slap someone"""
+        async with self.bot.session.get(
+                url="https://api.waifu.pics/sfw/slap"
+        ) as resp:
+            await ctx.send(
+                embed=discord.Embed(
```
Comment (Robin5605) [inline src/cogs/fun.py:150]: Should be `disnake.Embed(...)` to maintain consistency

File: src/cogs/fun.py [hunk 2]
```diff
@@ -141,18 +141,17 @@ async def kiss(self, ctx: Context, member: Member = None):
...
-        embed2 = Embed(
-            title=f"{ctx.author.mention} has slapped {member.name}"
-        ).set_image(
-            url="https://media.tenor.co/videos/318d19d23b24c54ab51cacf5ef4bfccf/mp4"
-        )
-
-        await ctx.send(embed=embed2)
+    async def slap(self, ctx, member: Member = None):
+        """slap someone"""
+        async with self.bot.session.get(
```
Comment (Robin5605) [inline src/cogs/fun.py:146]: The bot instance doesn't have a session variable like @Yat-o pointed out, though we are working on reusing the `aiohttp.ClientSession()` instance.

**Review by Robin5605 — CHANGES_REQUESTED** [review]
Please resolve the following issues I have outlined in the comments.

**Comment by Pandabweer** [comment]
Denied PR because of lack of attention, i suggest finishing on PR before opening a new one.

**Pull request closed without merging**